lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -157,9 +157,8 @@ static bool isTppUnaryOp(linalg::GenericOp linalgOp) {
 // Return true if the linalg.generic can be mapped to a tpp.add.
 bool isTppAdd(linalg::GenericOp linalgOp, SmallVectorImpl<Value> *operands) {
   using namespace tpp::structured_match;
-  auto addMatcher =
-      StructuredOpMatcher::make<linalg::GenericOp>()
-          .region(MatchOne(0), WithSingleOp<arith::AddFOp>(operands));
+  auto addMatcher = StructuredOpMatcher::make<linalg::GenericOp>().region(
```

Comment:
avoid formatting only changes as this conflicts with other in-flight PR.

Reply:
We should really add some hooks to enforce consistent formatting.

Reply:
#583

Reply:
Fixed formatting